packages/build-tools/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@vertexvis/build-tools",
-  "version": "0.8.4",
+  "version": "0.8.4-3",
```

Comment:
Is this version accurate?

Reply:
It'll get replaced on the release I'm about to do.